imap_processing/cdf/config/imap_ialirt_l1_variable_attrs.yaml

```
@@ -0,0 +1,370 @@
+default_attrs: &default
+  # Assumed values for all variable attrs unless overwritten
+  DEPEND_0: epoch
+  DISPLAY_TYPE: time_series
+  FILLVAL: -9223372036854775808
+  FORMAT: I12
+  VALIDMIN: -9223372036854775808
+  VALIDMAX: 9223372036854775807
+  VAR_TYPE: data
+  UNITS: " "
+
+component:
+  CATDESC: Velocity components (vx, vy, vz)
```

Component is a direction, the velocity could have components, but a component shouldn't reference velocity.
```suggestion
component:
  CATDESC: Component direction (x, y, z)
```